docs/releases/5.0.txt

```diff
@@ -634,6 +656,7 @@ Miscellaneous
 
 .. _`oracledb`: https://oracle.github.io/python-oracledb/
 
+
```

Comment:
Remove all new unrelated blank lines, and move this to the `5.1.txt` release notes.

Reply:
Sure, coming up 🤓